SU2_CFD/include/drivers/CSinglezoneDriver.hpp

```diff
@@ -98,6 +98,12 @@ class CSinglezoneDriver : public CDriver {
    */
   virtual bool Monitor(unsigned long TimeIter);
 
+  /*!
+   * \brief gets Convergence on physical time scale
+   * \param none
+   */
+  virtual bool GetTimeConvergence() const;
```

Comment:
What are the implications for multizone problems?

Reply:
At the moment, the convergence tools only work for single-zone problems, however it should be easy to extend it. I will speak about it with @talbring.

Reply:
Ok. If you want to merge the PR before that is done please add an error message or warning somewhere if someone tries to use the feature in multizone problems.

Reply:
I've added an error message for this case.